dev-portal/src/components/Feedback.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Button, Form, Modal, TextArea, Message } from 'semantic-ui-react'
+import { submitFeedback } from 'services/feedback'
+
+const initialState = {
+  message: '',
+  formOpen: false,
+  feedbackSubmitted: false
+}
+
+class Feedback extends React.PureComponent {
+  constructor(props) {
+    super(props)
+
+    this.state = initialState
+  }
+
+  openModal = () => {
+    this.setState({
+      formOpen: true
+    })
+  }
+
+  closeModal = () => {
+    this.setState(initialState)
+  }
+
+  handleSubmit = () => {
+    const { message } = this.state
+    submitFeedback(message).then(() => {
+      this.setState({
+        feedbackSubmitted: true
+      })
+    })
+  }
+
+  render() {
+    const { 
+      message,
+      formOpen,
+      feedbackSubmitted
+    } = this.state
+    
+    return (
+      <Modal 
+        closeIcon
+        centered={false}
+        size='tiny' 
+        open={formOpen}
+        onClose={this.closeModal}
+        trigger={
+          <div style={{ 
+            display: 'block', 
+            position: 'fixed', 
+            top: '50%', 
+            right: '-62px', // Adjusted for rotation
+            transform: 'rotate(90deg)'
+          }}>
+            <Button style={{
+              borderTopLeftRadius: '0',
+              borderTopRightRadius: '0'
+            }} onClick={this.openModal}><i className="envelope outline icon"></i>Got an opinon?</Button>
+          </div>
```

Comment:
Do you need this `div`? Can the `Button` be fixed/rotated?

Reply:
Probably, but I think it's much easier to read this way tbh. The button is taking care of its corners, while the container is taking care of the position